sklearn/linear_model/tests/test_ridge.py

```diff
@@ -1494,11 +1514,128 @@ def test_ridge_sag_with_X_fortran():
     Ridge(solver="sag").fit(X, y)
 
 
+@pytest.mark.parametrize("solver", ["auto", "lbfgs"])
+@pytest.mark.parametrize("fit_intercept", [True, False])
+@pytest.mark.parametrize("alpha", [1e-3, 1e-2, 0.1, 1.0])
+def test_ridge_positive_regression_test(solver, fit_intercept, alpha):
+    X = np.array([[1, 2], [3, 4], [5, 6], [7, 8]])
+    b = np.array([1, -10])
```

Comment:
can you call this coef. b suggests it's bias ie intercept

Reply:
same below

Reply:
Fixed.